docs/apib/node_bmc.apib

```diff
@@ -0,0 +1,15 @@
+FORMAT: 1A
+
+# XClarity Nodes
+
+# PUT /nodes/6B538D351B8211E193F65CF3FC6E4030/bmc
```

Comment:
@AndreyMenezes I tested this again this morning.  There is no JSON response to the PUT request for these operations.